client/processed_items_report/views/processed-items-report-panel.html

```diff
@@ -29,8 +28,10 @@
                 </div>
             </div>
 
-            <div ng-if="selectedUser">
-                {{ selectedUser.display_name }}<i class="icon-close-small" ng-click="removeUser()"></i>
+            <div ng-if="selectedUser" 
```

Comment:
You can use here selectedUsers array: selectedUsers.length; selectedUser is not needed and can be removed completely.